pkg/config/metadata/user_info.go

```diff
@@ -18,6 +19,11 @@ type UserInfo struct {
 func (self *UserInfo) Call(ctx common.AuthContext) (interface{}, error) {
 	// find oidc config and the userinfo endpoint
 	idConfig, _ := ctx.FindIdentityByName(self.OIDC)
+
+	if idConfig == nil {
+		return nil, fmt.Errorf("Null OIDC object for config %v. Skipping related UserInfo metadata.", self.OIDC)
```

Comment:
we should propagate a logger here

Reply:
Agreed. Out of scope of this PR though.

I was planning to work on the User Info evaluator anyway. Not happy with how it works now.

Thanks.